Description: Remove the build date from the manifest to make the build reproducible
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/build.xml
+++ b/build.xml
@@ -237,8 +237,6 @@
     <echo file="${build.dir}/info.txt">Manifest-Version: 1.0
 Sealed: false
 HttpUnit-Version: ${version}
-Build-Date: ${TODAY}
-Build-Time: ${TSTAMP}
 </echo>
     <jar jarfile="${lib.dir}/${name}.jar" manifest="${build.dir}/info.txt">
         <manifest>
